docs/scenario_editor/roads.md

```diff
@@ -27,6 +27,8 @@ OpenDrive can be loaded from file by:
 
 * Providing the path to an `.xodr` file into the "Filename" field and clicking "Load"
 
+* If "Generate traffic control assets?" is checked, the traffic control assets such as light poles, stop signs, and yield signs will be included, otherwise it will just spawn the road and traffic control devices. 
```

Comment:
The question mark is removed per your suggestion in the PR. Should read `Generate Traffic Control Assets`.